runtime/lua/vim/lsp/health.lua

```diff
@@ -111,8 +111,8 @@ local function check_watcher()
     watchfunc_name = 'libuv-watch'
   elseif watchfunc == vim._watch.watchdirs then
     watchfunc_name = 'libuv-watchdirs'
-  elseif watchfunc == vim._watch.inotifywait then
-    watchfunc_name = 'inotifywait'
+  elseif watchfunc == vim._watch.inotify then
+    watchfunc_name = 'inotify'
```

Comment:
👍 this aligns with the change in https://github.com/neovim/neovim/commit/55e4301036bb938474fc9768c41e28df867d9286 , which changed the program name but not the function name. And I think `inotify` is still a reasonable function name.